packages/story-editor/src/components/panels/document/taxonomies/shared.js

```diff
@@ -17,9 +17,24 @@
 /**
  * External dependencies
  */
+import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { Headline, THEME_CONSTANTS } from '@web-stories-wp/design-system';
 
+export const TaxonomyPropType = PropTypes.shape({
+  capabilities: PropTypes.object,
+  description: PropTypes.string,
+  hierarchical: PropTypes.bool,
+  labels: PropTypes.object,
+  name: PropTypes.string.isRequired,
+  rest_base: PropTypes.string.isRequired,
+  show_cloud: PropTypes.bool,
```

Comment:
We can change the name of these variables when we extract it from the REST API. Might make it cleaner down the line.

Reply:
I think that's a good idea. Added a new issue for this! https://github.com/google/web-stories-wp/issues/9057. Feel free to update it with anything you think fits.